app/e2e-tests/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "e2e-tests",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {},
```

Comment:
please add scripts for running tests

Reply:
this is coming in a follow up PR after this is merged in

Reply:
done!